RealmSwift/Migration.swift

```diff
@@ -203,7 +204,7 @@ extension Migration {
     public func enumerate(_ objectClassName: String, _ block: MigrationObjectEnumerateBlock) { }
 
     @available(*, unavailable, renamed:"createObject(ofType:populatedWith:)")
-    public func create(_ className: String, value: AnyObject = [:]) -> MigrationObject {
+    public func create(_ className: String, value: AnyObject = [:] as NSDictionary) -> MigrationObject {
```

Comment:
I figure we ought not change the APIs that are marked unavailable, no? They're only there to assist in migration.